### PR TITLE
Gt 364 Adobe Analytics

### PR DESCRIPTION
**Pull request by ryancarlson**

### CODE REVIEW

File: library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java [hunk 1]
```diff
@@ -42,8 +61,12 @@
     public static final String CATEGORY_CONTENT_EVENT = "Content Event";
     private Tracker mTracker = null;
 
+    private WeakReference<Activity> activity;
```
Comment (frett) [inline library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java]: Android naming convention for properties is:
`private WeakReference<Activity> mActivity`

File: library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java [hunk 2]
```diff
@@ -42,8 +61,12 @@
...
+    private String previousScreenName;
```
Comment (frett) [inline library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java]: `mPreviousScreenName`

**Comment by frett** [comment]
I also just noticed that when you moved the config json to the analytics module, you didn't put it in an assets folder

**Comment by ryancarlson** [comment]
Good catch on the config and folder

**Comment by ryancarlson** [comment]
@frett made the suggested changes... waiting on travis to confirm checkstyles. Anything else you would like to see?

**Pull request merged**